internal/store/alerts.go

```diff
@@ -26,6 +26,15 @@ type AlertDescriber interface {
 	Alert(string, string) (*atlas.Alert, error)
 }
 
+type AlertLister interface {
+	Alerts(string, *atlas.AlertsListOptions) (*atlas.AlertsResponse, error)
+}
+
+type AlertsStore interface {
```

Comment:
[qa] I do not why if I do not create the interface **AlertsStore**, `mongocli atlas alert describe alertID` seems using the wrong interface (it returns a list instead of one alert)